### PR TITLE
Fix a few FRD commands and minor consistency changes

### PR DESCRIPTION
**Pull request by joel16**
Tested these on hardware and they are working fine now.

### CODE REVIEW

File: libctru/include/3ds/services/frd.h [hunk 1]
```diff
@@ -5,9 +5,9 @@
 #pragma once
 #include <3ds.h>
 
-#define FRIEND_SCREEN_NAME_SIZE 0x16 // 11 (0x16 because UTF-16)
-#define FRIEND_COMMENT_SIZE 0x22     // 16 (0x21 because UTF-16 + null character)
-#define FRIEND_LIST_SIZE 0x64        // 100 (Max. number of friends)
+#define FRIEND_SCREEN_NAME_SIZE 0x16  ///< 11 (0x16 because UTF-16)
+#define FRIEND_COMMENT_SIZE 0x22      ///< 16 (0x21 because UTF-16 + null character)
```
Comment (mtheall) [inline libctru/include/3ds/services/frd.h]: Doc doesn't match value

File: libctru/include/3ds/services/frd.h [hunk 2]
```diff
@@ -242,7 +237,7 @@ Result FRD_AttachToEventNotification(Handle event);
  * @param size Number of events
  * @param recievedNotifCount Number of notification reccieved.
  */
-Result FRD_GetEventNotification(NotificationEvent *event, u32 size, u32 *recievedNotifCount); 
+Result FRD_GetEventNotification(NotificationEvent *event, size_t size, u32 *recievedNotifCount);
```
Comment (fincs) [inline libctru/include/3ds/services/frd.h]: Revert back to u32

File: libctru/include/3ds/services/frd.h [hunk 3]
```diff
@@ -170,59 +169,55 @@ Result FRD_GetMyFavoriteGame(u64 *titleId);
...
+ * @brief Get the current user's friends' favourite game.
  * @param desc Pointer to write Game Description data to.
- * @param keys Pointer to the friend key list.
- * @param offset Starting offset.
- * @param count Entry count.
+ * @param friendKeyList Pointer to FriendKeys,
+ * @param size Number Of FriendKeys.
  */
-Result FRD_GetFriendFavouriteGame(GameDescription *desc, const FriendKey *keys, u32 offset, u32 count);
+Result FRD_GetFriendFavouriteGame(GameDescription *desc, const FriendKey *friendKeyList, size_t size);
```
Comment (fincs) [inline libctru/include/3ds/services/frd.h]: Revert back to "u32 count"

File: libctru/include/3ds/services/frd.h [hunk 4]
```diff
@@ -170,59 +169,55 @@ Result FRD_GetMyFavoriteGame(u64 *titleId);
...
- * @brief Gets the current user's friend key list
+ * @brief Gets the current user's friend key list.
  * @param friendKeyList Pointer to write the friend key list to.
  * @param num Stores the number of friend keys obtained.
  * @param offset The index of the friend key to start with.
- * @param count Number of entries to read (max is FRIEND_LIST_SIZE).
+ * @param size Size of the friend key list. (FRIEND_LIST_SIZE)
  */
-Result FRD_GetFriendKeyList(FriendKey *friendKeyList, u32 *num, u32 offset, u32 count);
+Result FRD_GetFriendKeyList(FriendKey *friendKeyList, size_t *num, size_t offset, size_t size);
```
Comment (fincs) [inline libctru/include/3ds/services/frd.h]: Revert back to "u32 *num, u32 offset, u32 size"

File: libctru/include/3ds/services/frd.h [hunk 5]
```diff
@@ -170,59 +169,55 @@ Result FRD_GetMyFavoriteGame(u64 *titleId);
...
 /**
- * @brief Gets whether a friend code is included on the user's friend list.
- * @param friendCode Friend code to check.
- * @param isFromList Pointer to output the result to.
+ * @brief Gets whether a friend key is included in the current user's friend list.
+ * @param friendKeyList Pointer to a list of friend keys.
+ * @param isFromList Pointer to a write the friendship status to.
  */
-Result FRD_IsIncludedInFriendList(u64 friendCode, bool *isFromList);
+Result FRD_IsFromFriendList(FriendKey *friendKeyList, bool *isFromList);
```
Comment (fincs) [inline libctru/include/3ds/services/frd.h]: Rename to FRD_IsInFriendList

**Pull request merged**